R/heatmap.R

```diff
@@ -118,6 +119,7 @@ gggrob <- function(p, void = TRUE, nolegend = TRUE) {
     colors <- palette_this(glevels, palette = palette, palcolor = palcolor)
     if (!is.null(split_by)) {
         x <- x %>% unite("..split", split_by, group_by, remove = FALSE)
+        names(colors) <- names(split(x, x[["..split"]]))
```

Comment:
Should the same strategy be applied when palettle and palcolor are passed to the subplot?